Reed/Tokenizer/Tokenizer.swift

```diff
@@ -127,4 +133,22 @@ class Tokenizer {
         }
         return definitions
     }
+    
+    func makeFurigana(mecabWordNodes: [MecabWordNode], concatenatedToken: MecabWordNode, deinflectionResult: DeinflectionResult, definition: DictionaryDefinition) -> [Furigana] {
+        // Because fetching readings from JMDict is expensive, use JMDict's furigana only when:
+        // - More than two MecabWordNodes are concatenated (Otherwise, MeCab's furigana should be correct)
+        // - No deinflection applied (JMDict only knows how to read words in their dictionary forms)
+        // - The text doesn't contain kanji
```

Comment:
Do you mean if the text _does_ contain kanji? I don't think we need furigana for non-kanji terms.

Reply:
Yes!!!!!